miso-web/src/main/webapp/scripts/list_note.js

```diff
@@ -0,0 +1,208 @@
+ListTarget.note = (function () {
+  return {
+    name: "Notes",
+    createUrl: function (config, entityId) {
+      var id = config.sampleId || config.libraryId || entityId;
+      var entityType = config.entityType;
+
+      if (!id || !entityType) {
+        console.error("Error: Missing entity ID or type for notes");
+        return null;
+      }
+
+      return "/rest/notes/" + entityType + "/" + id + "/dt?includeRelated=true";
+    },
+    getQueryUrl: function () {
+      return "/rest/notes/search";
+    },
+    createBulkActions: function (config, entityId) {
+      return [
+        {
+          name: "Delete",
+          action: function (items) {
+            if (items.length === 0) return;
+            var id = config.sampleId || config.libraryId || entityId;
+            var entityType = config.entityType;
+            if (!id || !entityType) {
+              Utils.showOkDialog("Error", ["Cannot determine which entity these notes belong to."]);
+              return;
+            }
+            var currentEntityNotes = items.filter(function (note) {
+              var typeMatches =
+                note.entityType.toLowerCase().indexOf(entityType.toLowerCase()) !== -1;
+              var idMatches = String(note.entityId) === String(id);
+
+              return typeMatches && idMatches;
+            });
+
+            if (currentEntityNotes.length === 0) {
+              Utils.showOkDialog("Error", ["Cannot delete notes from parent entities."]);
+              return;
+            }
+            var lines = ["Are you sure you want to delete these notes? This cannot be undone."];
+            var ids = [];
+            jQuery.each(currentEntityNotes, function (index, note) {
+              lines.push("* " + note.text);
+              ids.push(note.id);
+            });
+            var url = "/rest/notes/" + entityType + "/" + id + "/" + ids.join(",");
```

Comment:
This is a weird URL. IDs to delete should be provided in the request body like other bulk delete requests. See `createBulkDeleteAction` in `list.js` for example. That function won't work here, but this should follow the same pattern.